### ISSUE TITLE
workflow testing and development

### ISSUE THREAD

**Issue by bgruening** [usegalaxy-au/elixir-biocommons-colab#12]
* add more tests to GTN workflows
* port more workflows to Galaxy

**Comment by bgruening** [comment]
@annasyme, @GraceAHall

**Comment by hexylena** [comment]
Would be super cool to see! Here's the GTN FAQ on adding WF tests. https://training.galaxyproject.org/training-material/faqs/gtn/gtn_workflow_testing.html We don't have a nice page currently listining which ones are missing it, but I can generate that if it's useful.

**Comment by bgruening** [comment]
@mthang maybe you are also interested in this?

**Comment by AnnaSyme** [comment]
Would it be good to convert this info about adding to the IWC into a GTN tutorial? not sure https://github.com/AnnaSyme/iwc/blob/assembly-wfs/workflows/README.md#adding-workflows

**Comment by AnnaSyme** [comment]
Should we delay working on this one for a few weeks?

**Comment by hexylena** [comment]
Sure, that'd be fine @AnnaSyme! I'm not sure where the IWC would prefer to have their docs (github makes sense since it's a bit technical to contribute I guess) but of course we'd be happy to have tutorials on e.g. adding a workflow to the IWC in the GTN (pinging @mvdbeek) if that would be useful to their users.

**Comment by supernord** [comment]
@uwwint

**Comment by mvdbeek** [comment]
Yeah, we talked about this, I'm fine with having links on github point at the GTN ... this is where people find tutorials these days, so the discoverability is better there.

**Comment by bgruening** [comment]
Annas IWC workflow is merged: https://github.com/galaxyproject/iwc/pull/200
